starter-code/src/viking.js

```diff
@@ -1,11 +1,70 @@
 // Soldier
-function Soldier() {}
+function Soldier(healthArg, strengthArg) {
+    this.health = healthArg;
+    this.strength = strengthArg;
+}
+
+Soldier.prototype.attack = function() {
+    return this.strengthArg;
```

Comment:
`this.strengthArg`doesn't exist on the `Soldier` constructor. What you have is `this.health` and `this.strength`